src/ucp/wireup/address.c

```diff
@@ -504,20 +505,22 @@ ucp_address_packed_size(ucp_worker_h worker,
             /* device md_index */
             md_index   = worker->context->tl_rscs[dev->rsc_index].md_index;
             /* md index (+flags) can take 2 bytes with address version 2 */
-            value_size = ucp_address_packed_value_size(md_index, md_mask,
-                                                       addr_version);
+            value_size = ucp_address_packed_value_size(
+                    md_index, md_mask, addr_version,
+                    worker->context->tl_rscs[dev->rsc_index].tl_rsc.dev_name);
```

Comment:
maybe cache the (const) pointer to `worker->context->tl_rscs[dev->rsc_index]` in a local var? it's used several times in this loop..

Reply:
fixed